client/lib/wpcom-undocumented/lib/undocumented.js

```diff
@@ -1127,6 +1127,15 @@ Undocumented.prototype.readA8C = function( query, fn ) {
 	return this.wpcom.req.get( '/read/a8c', query, fn );
 };
 
+Undocumented.prototype.readConversations = function( query, fn ) {
```

Comment:
i think we need a file like this for data-layer.  just full of http action creators instead of actual network calls